### ISSUE TITLE
Feature request: runtime option to print detailed kernel compilation error

### ISSUE THREAD

**Issue by khaotik** [Theano/libgpuarray#439]
The ability to compile GPU kernel is exposed in API. However, in order to see compilation error, I had to rebuild the entire libgpuarray in `DEBUG` mode. For development outside libgpuarray itself (e.g. in `theano/gpuarray`), I think it is better to have a runtime option to control verbosity.

**Comment by abergeron** [comment]
I agree with this.  I don't know when I'll have time to look into it though.
